server/middleware/isAdmin.js

```diff
@@ -0,0 +1,10 @@
+const { boomify } = require('../utilis');
+
+const isAdmin = (req, res, next) => {
+  if (req.role !== 'admin') {
+    next(boomify(401, "you're unautherized"));
```

Comment:
![image](https://user-images.githubusercontent.com/32321796/113263421-6cc04d80-92da-11eb-87c9-23b06ac8787e.png)
please edit the miss-spelled word `unautherized` to be `unauthorized`

- and make sure [`Code Spell Checker`](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) is installed in your vs Code to detects any typos.